packages/upload-api/test/helpers/context.js

```diff
@@ -47,7 +47,7 @@ export const createContext = async (
     storefront: { filecoinSubmitQueue, pieceOfferQueue },
   } = getFilecoinQueueImplementations(queuedMessages)
   const {
-    storefront: { pieceStore, receiptStore, taskStore },
+    storefront: { pieceStore },
```

Comment:
should we not do in same PR `filecoin-api` and `upload-api` refactor for this? This would make us need to update both infras as well, and perhaps better one at a time?

Reply:
See https://github.com/w3s-project/w3up/blob/main/packages/filecoin-api/src/storefront/api.ts#L61